__tests__/createAnimation.js

```diff
@@ -108,4 +119,18 @@ describe('createAnimation', () => {
       },
     });
   });
+
+  it('should return value from cache', () => {
+    const definition = {
+      from: {
+        opacity: 0,
+      },
+      to: {
+        opacity: 1,
+      },
+    }
+    const firstAnimation = createAnimation(definition)
+    const secondAnimation = createAnimation(definition)
+    expect(secondAnimation).toEqual(firstAnimation)
```

Comment:
This line should be `toBe` to test referential equality.

Reply:
Why did I forget bout that...